azure-pipelines.yml

```
@@ -66,7 +66,7 @@ jobs:
       displayName: Install dependencies
     - script: |
         cd hypothesis-python
-        pytest
+        pytest -n auto
```

```suggestion
        pytest --numprocesses auto
```

Let's spell out the name of the option for super-duper clarity (because I couldn't remember what this option does, and it's not in the default pytest options).